CMakeLists.txt

```diff
@@ -329,6 +333,27 @@ if(SDL3MIXER_CMD)
     endif()
 endif()
 
+if(SDL3MIXER_SNDFILE)
```

Comment:
Please Add an if-branch for `SDL3MIXER_VENDORED`, that should fail. (see `SDL3MIXER_MOD_MODPLUG`)